### DISCUSSION TITLE
BT / Acaia with Artisan & Monterey 12.0.1

### DISCUSSION THREAD

**Discussion by Skjagodski** [artisan-roaster-scope/artisan#730] [General]
After upgrading my Mac to Monterey, Artisan works fine, just like on the earlier OS versions.
The BT connect to the Acaia scale seems not to be active though. 
"---" appears instead of the weight.
Maybe I overlooked / skipped the pop up question to allow BT connections for Artisan.
Is there a menu where I can activate / allow the BT connect?

**Comment by Skjagodski** [comment]
<img width="653" alt="Schermafbeelding 2021-11-23 om 13 51 07" src="https://user-images.githubusercontent.com/94895776/143027166-431a22c6-5e60-4494-8133-f219939cd9dc.png">

I checked and Artisan has access to BT...

**Reply by MAKOMO** [reply]
Just checked Artisan v2.4.6 on Monterey and it connects to my Acaia. Which scale are you using? Did you select "acaia" under Config >> Ports >> Scale?

**Comment by Skjagodski** [comment]
I use the big black one, the black Pearl. Also tried the little black Lunar. And the tiny silver Cinco.
I had nog changed the port settings, keeping the config from earlier Artisan installs.
I tried to select different port settings also..
<img width="555" alt="Schermafbeelding 2021-11-23 om 18 38 56" src="https://user-images.githubusercontent.com/94895776/143077095-46165dc6-0e20-44f8-8699-8e03bed00c01.png">
<img width="591" alt="Schermafbeelding 2021-11-23 om 18 39 49" src="https://user-images.githubusercontent.com/94895776/143077112-44498dac-861e-4e65-9efc-05452e97ca57.png">
.

**Comment by MAKOMO** [comment]
I am asking about the scale as the newer 2021 and the Pearl-S models will be supported only by the next Artisan release.

You might check the current continuous build and see if that works for you. If not, please activate debug logging by pressing OPTION+COMMAND (macOS) while clicking on the plus icon in the upper left corner (ALT+CONTROL on Windows), connect to the scale and add here the log file (click again the plus icon while holding just the OPTION rsp. the ALT key).

**Comment by MAKOMO** [comment]
The CommPort does not play a role for BLE communication. However, older Windows version are not supported. This should work from Windows 10 on.

**Comment by Skjagodski** [comment]
Thanks! I downloaded the install from
[https://github.com/artisan-roaster-scope/artisan/releases/tag/v2.4.6](https://github.com/artisan-roaster-scope/artisan/releases/tag/v2.4.6)
and after starting it, **the request to allow BT comms** did come on screen, I OK'd it and the scale was recognized!
All good
<img width="272" alt="Schermafbeelding 2021-11-23 om 19 54 47" src="https://user-images.githubusercontent.com/94895776/143086624-f836f90a-1886-42f8-bba8-00b9e629454c.png">
.

**Comment by Skjagodski** [comment]
"The newer 2021 and the Pearl-S models will be supported only by the next Artisan release."

So the older Acaia models will no longer wirelessly communicate with future Artisan versions? It's not a problem, a good reason to upgrade my Acaia collection ;-)

**Reply by MAKOMO** [reply]
No, the next version of Artisan will support the full range of Acaia scales, old and new. The current version v2.4.6 only supports the original Lunar and Pearl.
